lib/src/config_provider/config.dart

```diff
@@ -215,363 +219,699 @@ class Config {
     }
   }
 
-  /// Validates Yaml according to given specs.
-  bool _checkConfigs(YamlMap map, Map<List<String>, Specification> specs) {
-    var result = true;
-    for (final key in specs.keys) {
-      final spec = specs[key];
-      if (checkKeyInYaml(key, map)) {
-        result = result && spec!.validator(key, getKeyValueFromYaml(key, map));
-      } else if (spec!.requirement == Requirement.yes) {
-        _logger.severe("Key '$key' is required.");
-        result = false;
-      } else if (spec.requirement == Requirement.prefer) {
-        _logger.warning("Prefer adding Key '$key' to your config.");
-      }
+  ConfigSpec _getRootSchema() {
+    return FixedMapConfigSpec(
```

Comment:
Remove `Fixed`, whether it allows extra keys or not should be a bool in the constructor.